### ISSUE TITLE
A game can be initialized with a given seed for the random number generator

### ISSUE THREAD

**Issue by swom** [tresinformal/game#288]
The following test should pass in game.cpp:
```c++
```

**Comment by Clemcarle** [comment]
We couldn't really understand two of the errors that popped at the bottom of the test. 
@swom, Richel said you should have a look again?

**Comment by swom** [comment]
Yes, my fault, bad writing. You need o e more empty round brackets after `g.get_rng()` like this
`g.get_rng()()`.
The idea behind the test is that the two random number generators, when seeded with the same seed, call the same number
